main.py

```diff
@@ -3,7 +3,12 @@
 
 from flask import Response
 from vitumob import app
+from mongoengine import connect
+from mongoengine import *
 
+connect('vitumob1')
```

Comment:
You need to check if you're in `localhost` and use this, if not use the cloud database URL.